Remove extra files from client assets folder (#1683)
codemagic-ci-cd/codemagic-docs: content/knowledge-white-label/white-label-scripts.md

       'https://api.codemagic.io/apps/<app-id>/variables'
{{< /highlight >}}

{{<notebox>}}

> **Note:** Use this command if you want to add a base64 encoded value. 

Reviewer: This might cause confusion. Which values should or shouldn't be base64 encoded? I would mention the .keystore. In the case of a .env file, I would say that it's encoded to preserve the file formatting.

       'https://api.codemagic.io/apps/<app-id>/variables'
{{< /highlight >}}

{{<notebox>}}
**Note:** 

> Sometimes you might want to store files e.g. **Android keystores**, or **.env** files. You can pass them as a base64 encoded value like this:

Reviewer: ```suggestion
Files such as **Android keystores**, or **.env** files should be base64 encoded and can be passed like this:
```
